src/Error.php

```diff
@@ -38,14 +40,14 @@ class Error
      *
      * @var string
      */
-    protected $message;
+    protected $message = '';
 
     /**
      * @param  string $message
      * @param  int $code
      * @param  mixed $data
      */
-    public function __construct($message = null, $code = self::ERROR_OTHER, $data = null)
+    public function __construct(?string $message = '', int $code = self::ERROR_OTHER, $data = null)
```

Comment:
Probably `$message` shouldn't be nullable.

Reply:
Agreed. Change will be pushed shortly.